docs/docs/reactions/projections.md

```diff
@@ -67,35 +64,35 @@ async function handler(streamName: string, events: ITimelineEvent[]) {
   if (statements.length === 0) return
   const client = await pool.connect()
   try {
-    // CAUTION: do not lower the isolation level unless you know what you're doing 
-    await client.query('BEGIN TRANSACTION ISOLATION LEVEL SERIALIZABLE')
+    // CAUTION: do not lower the isolation level unless you are Kimi Raikkonen
```

Comment:
I guess if you know you know but https://www.google.com/search?q=raikkonen+quote gives a good result at this moment in time for me!

Reply:
![image](https://github.com/equinox-project/equinox-js/assets/3721521/2db9522c-fa3a-4afb-b862-44c01e47b119)